params/config/sim_config.yaml

```diff
@@ -17,7 +17,7 @@ ttl_choices: [50]
 
 # Optional: Trace file trace relative to the CWD.
 # Until values start in the trace file, the defaults from this file are used
-# trace_path: params/traces/default_trace.csv
+trace_path: params/traces/default_trace.csv
```

Comment:
Could you keep this commented out by default? For consistency with the examples in the readme. Thanks!

Reply:
I already applied the change myself :)